acceptance/bundle/sync/dryrun/script

```diff
@@ -0,0 +1,18 @@
+mkdir "project-folder" "ignored-folder" "ignored-folder/folder1"
+touch "project-folder/app.yaml" "project-folder/app.py" "project-folder/query.sql"
+touch "ignored-folder/script.py" "ignored-folder/folder1/script.py"
+cat > .gitignore << EOF
+ignored-folder/
+script
+output.txt
+repls.json
+EOF
```

Comment:
Would it not be more natural to prepare this structure and commit it to git?

You would need to come up with another name for .gitignore (e.g. test-gitignore) and do 'mv test-gitignore .gitignore' before your test.